src/date.cpp

```diff
@@ -25,22 +25,25 @@
 
 #include "safeguards.h"
 
-#define M(a, b) ((a << 5) | b)
+static uint16 PackMonthDateHelper(uint8 a, uint8 b)
+{
+	return (a << 5) | b;
+}
+
 static const uint16 _month_date_from_year_day[] = {
```

Comment:
This is not really an improvement. It would be better to have a 2 value struct and just use initialization to fill it, skipping stuffing it into a single 16 bit value.

Reply:
I'm going to leave this one alone for now...